src/components/consent-notice.js

```diff
@@ -66,12 +60,12 @@ export default class ConsentNotice extends React.Component {
             <div className="cn-body">
                 <p>
                     {t(['consentNotice', 'description'], {purposes: <strong>{purposesText}</strong>})}
-                    <a href="#" onClick={this.showModal}>{t(['consentNotice', 'learnMore'])}...</a>
                 </p>
                 {changesText}
                 <p className="cn-ok">
-                    <button className="cm-btn cm-btn-sm cm-btn-success" type="button" onClick={this.saveAndHide}>{t(['ok'])}</button>
-                    <button className="cm-btn cm-btn-sm cm-btn-danger cn-decline" type="button" onClick={this.declineAndHide}>{t(['decline'])}</button>
+                    <button className="cm-btn cm-btn-success" type="button" onClick={this.saveAndHide}>{t(['ok'])}</button>
+                    <button className="cm-btn" type="button" onClick={this.declineAndHide}>{t(['decline'])}</button>
+                    <button className="cm-btn cm-btn-info" type="button" onClick={this.showModal}>{t(['consentNotice', 'learnMore'])}</button>
```

Comment:
How about "Settings" instead? Many consent banners (those without actual functionality) often use "Learn more" or something similar for a link to the privacy page.

Reply:
True, but that sounds like a different issue (including updates for all languages).